webpack/scenes/ContentViews/Create/__tests__/createContentView.test.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { renderWithRedux, patientlyWaitFor, fireEvent } from 'react-testing-lib-wrapper';
+
+import { nockInstance, assertNockRequest } from '../../../../test-utils/nockWrapper';
+import api from '../../../../services/api';
+import CreateContentViewForm from '../CreateContentViewForm';
+
+const cvCreateData = require('./contentViewCreateResult.fixtures.json');
+
+const cvCreatePath = api.getApiUrl('/content_views');
+
+const setModalOpen = jest.fn();
+
+const createDetails = {
+  name: '1232123',
+  label: '1232123',
+  description: '',
+  composite: false,
+  dependencies: false,
+  auto_publish: false,
+  import_only: false,
+};
+
+const createdCVDetails = { ...cvCreateData };
+
+const form = <CreateContentViewForm setModalOpen={setModalOpen} />;
+
+test('Can save content view from form', async (done) => {
+  const createscope = nockInstance
+    .post(cvCreatePath, createDetails)
+    .reply(201, createdCVDetails);
+  const { queryByText, getByLabelText } = renderWithRedux(form);
+  expect(queryByText('Description')).toBeInTheDocument();
+
+  fireEvent.change(getByLabelText('input_name'), { target: { value: '1232123' } });
+
+  await patientlyWaitFor(() => { expect(getByLabelText('input_label')).toHaveAttribute('value', '1232123'); });
+
+  getByLabelText('create_content_view').click();
+
+  assertNockRequest(createscope, done);
+});
+
+test('Form closes itself upon save', async (done) => {
```

Comment:
nitpick - you are testing the save functionality here, I would argue that is the bigger part you are testing, so could call this `Content view create call is sent to API and modal closes upon save`